__tests__/unit/media-play-button.spec.js

```diff
@@ -0,0 +1,145 @@
+import { spy } from 'sinon';
+import { fixture, expect } from '@open-wc/testing';
+import { MediaUIAttributes, MediaUIEvents } from '../../src/js/constants';
+import { verbs } from '../../src/js/labels/labels';
+import '../../src/js/media-play-button';
+
+describe('<media-play-button/>', () => {
+  it('passes the a11y audit', async () => {
+    const el = await fixture(`<media-play-button></media-play-button>`);
+    await expect(el).shadowDom.to.be.accessible();
```

Comment:
wtf, you can just do `to.be.accessible()`? Wow.

Reply:
yeah. it's not perfect, but it's a good audit to have. It's built on axe core (https://www.npmjs.com/package/chai-a11y-axe)